rslib/src/storage/card/mod.rs

```diff
@@ -872,6 +883,9 @@ fn review_order_sql(order: ReviewCardOrder, timing: SchedTimingToday, fsrs: bool
         ReviewCardOrder::RetrievabilityDescending => {
             build_retrievability_clauses(fsrs, timing, SqlSortOrder::Descending)
         }
+        ReviewCardOrder::RelativeOverdueness => {
+            vec![ReviewOrderSubclause::RelativeOverdueness { timing }]
```

Comment:
```suggestion
            build_retrievability_clauses(fsrs, timing, SqlSortOrder::Ascending)
```
The test passes if you do this

Reply:
I think creating a separate path for SM-2 is better for performance reasons as well as to avoid fighting against the nuances of the testing mechanism. Thanks for the help.